src/libraries/System.Net.Security/src/System/Net/Security/SslClientAuthenticationOptions.cs

```diff
@@ -73,5 +73,11 @@ public SslProtocols EnabledSslProtocols
         /// Use extreme caution when changing this setting.
         /// </summary>
         public CipherSuitesPolicy? CipherSuitesPolicy { get; set; }
+
+        /// <summary>
+        /// Specifies X509ChainPolicy to use for remote certificate
```

Comment:
Following the style from https://github.com/dotnet/dotnet-api-docs/wiki/Summary%3A-Property

```suggestion
        /// Gets or sets an optional customized policy for remote certificate
```